common/logger/logger.go

```diff
@@ -174,5 +176,9 @@ func GetLogger() Logger {
 
 // getLogWriter get Lumberjack writer by LumberjackConfig
 func (c *Config) getLogWriter() zapcore.WriteSyncer {
-	return zapcore.AddSync(c.LogRollingConfig)
+	if !c.LogStdout {
```

Comment:
The default is to print both to the console and to a file, which may be better to reduce user configuration。
such as:
`core := zapcore.NewCore(zapcore.NewConsoleEncoder(encoder), zapcore.NewMultiWriteSyncer(writer, zapcore.AddSync(os.Stdout)), logLevel)`